extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -163,5 +160,27 @@ amp-carousel .amp-carousel-button.amp-disabled {
   order: 100000000 !important;
   margin-right: -1px !important;
 }
-
 /* End SlideScroll */
+
+
+
+
+
+/* Begin ScrollableCarousel */
+.-amp-scrollable-carousel-container .amp-carousel-slide {
```

Comment:
now that we rely on this class can you add in the the test and make sure it gets added during buildcallback?